projects/canopy/src/lib/accordion/docs/guide.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Accordion/Guide" parameters={{ previewTabs: { canvas: { hidden: true } } }} />
+
+# Accordion
+
+<p class="standfirst">The accordion component reduces page clutter and cognitive load by letting users reveal sections of information as and when they need it.</p>
+
+![Accordion component example](docs/accordion/example.png)
+
+## Usage
+
+Use the accordion component to hide sections of content to make pages easier to scan.
+
+Information that is important or relevant to most users should not be hidden within this component and should be displayed on the page instead.
+
+Accordions work best for simple content and links but can support rich content (sound, video, imagery, etc), if required.
+
+Import the accordion module into your module:
+
+```js
+@NgModule({
+  ...
+  imports: [ ..., LgAccordionModule ],
+})
+```
+
+
+
+and in your HTML:
+
+```html
+<lg-accordion [headingLevel]="2">
```

Comment:
(a little nitpicky 😅) is the indentation meant to match this first line, or the rest?

Reply:
Updated 👍 haven't deployed it though!